workflow/pred_temperature_autoencoder.ipynb

```diff
@@ -0,0 +1,1298 @@
+{
```

Comment:
<p>Loss should be low, right? So why is the validation loss so high compared to the training loss, is this overfitting?</p><p>The loss for the validation data is very high after the first few epochs.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Yes, this is terribly overfitting...Simply because the data we have is far from sufficient. I didn't try hard to improve it since we will have era5 data, which will be (slightly) longer than the testing data. I plan further tune this model later when we use the new data. For now I can simply reduce the number of epoch, to make it less overfitting.</p>